src/oc/x.c

```diff
@@ -68,6 +77,7 @@ void x11_fast(int mode)
 }
 
 void x11flush(void) {
+ifnox_return
```

Comment:
nitpick: this can be indented as code statement?

Reply:
Yes. Notice that the macro has no ';'.   I suppose this will cause trouble for clang-format. Should the macro be written as
{ifnox_return if(!hoc_usegui) { return; }}
and
/**/ or possibly {;}
and a ';' be placed at the end of each invocation?

Reply:
I think that extra {} won’t change formatting. We can ident for now and once I apply clang-format we can review that.

Reply:
changed the macro name and indented.